oak-core/src/test/java/org/apache/jackrabbit/oak/plugins/index/property/strategy/ContentMirrorStoreStrategyTest.java

```diff
@@ -31,12 +31,14 @@
 
 import java.util.Collections;
 import java.util.HashSet;
+import java.util.List;
```

Comment:
please remove this unused import.

https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=1922&id=org.apache.jackrabbit%3Ajackrabbit-oak&open=AZQIRx8DbcZ1hbDMIuAh